src/viewport.js

```diff
@@ -107,7 +107,7 @@ $.Viewport = function( options ) {
         defaultZoomLevel:           $.DEFAULT_SETTINGS.defaultZoomLevel,
         minZoomLevel:               $.DEFAULT_SETTINGS.minZoomLevel,
         maxZoomLevel:               $.DEFAULT_SETTINGS.maxZoomLevel,
-        degrees:                    $.DEFAULT_SETTINGS.degrees,
+        initialDegrees:             $.DEFAULT_SETTINGS.degrees,
```

Comment:
We need to make sure that `options.degrees` still works for input to the constructor. Above this `$.extend` section, you should add `options.initialDegrees = options.degrees` and `delete options.degrees` to make that happen.